diff --git a/configure.ac b/configure.ac
index 247806d..5ba6d72 100644
--- a/configure.ac
+++ b/configure.ac
@@ -259,10 +259,10 @@ if test "$have_gtk" = "yes"; then
         PKG_CHECK_MODULES(LIBNOTIFY, [ libnotify >= 0.7.0 ])
         AC_SUBST(LIBNOTIFY_CFLAGS)
         AC_SUBST(LIBNOTIFY_LIBS)
-fi
 
-AM_PROG_VALAC([0.11])
-AC_SUBST(VAPIDIR)
+	AM_PROG_VALAC([0.11])
+	AC_SUBST(VAPIDIR)
+fi
 AM_CONDITIONAL(HAVE_VALAC, test x"$VALAC" != x)
 
 AC_PATH_PROG([XSLTPROC], [xsltproc])
